src/openrct2/core/Console.cpp

```diff
@@ -19,6 +19,7 @@ extern "C"
     #include "../platform/platform.h"
 }
 
+#include <android/log.h>
```

Comment:
Revert but create an issue for it, so it can be done later on? At this point it's probably best to get this humongous thing in.